homework/12_V/26_Hristiyan/homework-1/circles.py

```diff
@@ -0,0 +1,27 @@
+import math
+
+
+def circles(x1, y1, r1, x2, y2, r2):
+    d = math.sqrt((x1 - x2)**2 + (y1 - y2)**2)
+    # (5 - 1)** 2 + (4 - 1)**2 = 25
+    # d = 5
+    if d <= r1 - r2:
+        return "C2 is in C1"
+    elif d <= r2 - r1:
```

Comment:
if `d == r1 + r2 or d == |r1-r2|` then the two circles are touching. Add `touching` as another condition.